plugins/modules/open_iscsi.py

```diff
@@ -240,21 +239,22 @@ def target_login(module, target, portal=None, port=None):
                   ('node.session.auth.password', node_pass)]
         for (name, value) in params:
             cmd = [iscsiadm_cmd, '--mode', 'node', '--targetname', target, '--op=update', '--name', name, '--value', value]
-            module.run_command(cmd, check_rc=True)
+            rc, out, err = module.run_command(cmd, check_rc=check_rc)
```

Comment:
I'm not sure why you store `rc, out, err` here if `check_rc=False`, since you never use these values.

Reply:
Oops, i initially thought that these commands run exclusively
Thanks for noticing!